question/bank/managecategories/category.php

```diff
@@ -24,12 +24,12 @@
  */
 
 
-require_once("../config.php");
+require_once("../../../config.php");
```

Comment:
add a / in the beginning

Reply:
If we add  / in the beginning we need also `__DIR__` , done.